src/redux/services/fileEntry.ts

```diff
@@ -188,15 +190,24 @@ export function readFiles(
     );
   } else {
     files.forEach(file => {
+      let text;
       const filePath = path.join(folder, file);
       const fileEntryPath = filePath.substring(rootFolder.length);
-      const fileEntry = createFileEntry({fileEntryPath, fileMap, helmChartId: helmChart?.id});
+
+      const isDir = getFileStats(filePath)?.isDirectory();
+
+      if (!isDir) {
```

Comment:
maybe we should not read the content of files that are not "supported" or "included" ? based on `scanExcludes` and `fileIncludes`

Reply:
updated